ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,38 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# asanyarraya
+
+@handle_frontend_test(
+    fn_tree="numpy.asanyarray",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+)
+def test_numpy_asanyarray(
+    *,
+    dtype_and_x,
+    as_variable,
+    like,
```

Comment:
`like` and `order` are not defined in the decorator. Please define them using hypothesis strategies.

Reply:
Hello. I've done it. But if I'm wrong please point me to the right direction. It means I didn't understand the last review comment you made. I hope I'm correct this time. I've been on this for 3 weeks. Thanks for all your time and corrections. I appreciate.